src/dbnode/kvconfig/keys.go

```diff
@@ -49,4 +49,16 @@ const (
 	// ClientWriteConsistencyLevel is the KV config key for the runtime
 	// configuration specifying the client write consistency level
 	ClientWriteConsistencyLevel = "m3db.client.write-consistency-level"
+
+	// DocsLimit is the KV config key for the docs matched query limit.
+	// Settings in string form "{limit},{lookback}", e.g. "1000,15s".
+	DocsLimit = "m3db.limits.docs"
```

Comment:
can we just have a single limit proto stored in etcd? instead of a kv for each limit